.github/workflows/macos.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           - MODE: distcheck
           - MODE: qa-sh
             FLAT_ENV: CC=clang CXX=clang++ LD=clang++ QA_SANITIZER=address
-    runs-on: macos-11
+    runs-on: [macos-12, macos-14]
```

Comment:
I believe this does not work and would need to through the matrix above instead, similar to e.g.:

```console
# git grep -w os .github/workflows/autotools-cmake.yml | head -n4
.github/workflows/autotools-cmake.yml:          - os: macos-11
.github/workflows/autotools-cmake.yml:          - os: ubuntu-20.04
.github/workflows/autotools-cmake.yml:          - os: ubuntu-20.04
.github/workflows/autotools-cmake.yml:    runs-on: "${{ matrix.os }}"
```

It would explain why the CI fails to start at both…

- your https://github.com/dag-erling/libexpat/actions/runs/8921633615
- and https://github.com/libexpat/libexpat/actions/runs/8921633842?pr=868 here

…saying `Requested labels: macos-12, macos-14` which may be hard to fullfil by a single runner at the same time, unless it's about image availability.

PS: `runs-on` or `os` would both work well for a matrix entry name.

Reply:
I'll look into it. GitHub's documentation is sadly not very helpful.

Reply:
> I'll look into it. GitHub's documentation is sadly not very helpful.

It would be https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on . They do explain what's going on, but maybe not in ideal way.

Reply:
> I'll look into it.

Thanks! :+1